app/Http/Controllers/Articles/ArticlesController.php

```diff
@@ -33,11 +33,13 @@ public function index(Request $request)
             ->get();
         $moderators = User::moderators()->get();
         $canonical = canonical('articles', $request->only('sortBy', 'tag'));
+        $topSubmitters = User::mostSubmitsInLastDays(365)->take(5)->get();
```

Comment:
Rename this to `$topAuthors` and use that terminology in the subsequent views as well.